.github/workflows/test-docker.yml

```diff
@@ -47,8 +50,15 @@ jobs:
           docker exec hdk-build.${{ inputs.name }} chown -R ghrunner:ghrunner /_work/
           docker exec -u ghrunner hdk-build.${{ inputs.name }} tar -zxf /_work/build.tgz -C /_work/
 
-      - name: Test
-        run: docker exec -u ghrunner hdk-build.${{ inputs.name }} bash /_work/omniscidb/scripts/conda/test.sh
+      - name: Smoke test
+        if: inputs.name == 'cuda'
```

Comment:
Is this a cuda-only flow?

Reply:
this is a "self hosted" workflow; it applies now to two scenarios: CUDA tests and ASAN tests e g those we cannot run with github runners

Reply:
the strange way to pass "cuda" into a reusable workflow and query for it comes from two requirements/limitations:
1) Igor requests no cut&paste;
2) it was not possible (until recently) to call a function from another function

this results in the following approach

<common code>
<code for a cuda runner> if cuda
<code for an asan runner> if asan